Support Fetch Cilium Observe data to monitoring Cilium Service network traffic (#12393)
apache/skywalking: docs/en/setup/backend/backend-k8s-monitoring.md

2. Rover is a SkyWalking native eBPF agent to collect network Access Logs to support topology-aware and metrics 
   analysis. Meanwhile, due to the power of eBPF, it could profile running services written by C++, Rust, Golang, etc. 
   Read [Rover setup guide](./backend-k8s-monitoring-rover.md) for more details.

> 3. If Cilium is installed in Kubernetes, use Cilium Fetcher can collect network traffic data from services. 

Reviewer: Besides this, we should have a menu item for Cilium integration.

```suggestion
3. If Cilium is installed in Kubernetes, use Cilium Fetcher to collect network traffic data of services through Cilium Hubble APIs. 
```

Is this correct?

Author: Correct. And I have update the menu.
